decidim-participatory_processes/lib/decidim/participatory_processes/engine.rb

```diff
@@ -261,26 +261,36 @@ class Engine < ::Rails::Engine
           content_block.cell = "decidim/proposals/content_blocks/highlighted_proposals"
           content_block.settings_form_cell = "decidim/content_blocks/highlighted_elements_settings_form"
           content_block.public_name_key = "decidim.proposals.content_blocks.highlighted_proposals.name"
+          content_block.component_manifest_name = "proposals"
 
           content_block.settings do |settings|
             settings.attribute :order, type: :enum, default: "random", choices: %w(random recent)
+            settings.attribute :show_space, type: :boolean, default: true
           end
         end
 
         Decidim.content_blocks.register(:participatory_process_group_homepage, :highlighted_results) do |content_block|
```

Comment:
While reviewing #10573, i realised that we could add a check to see if the accountability module is defined.